src/app/examples/examples.routes.ts

```diff
@@ -36,6 +36,15 @@ import { PageAdvancedExampleComponent } from './page-example/advanced/page-advan
 import { ExamplesComponent } from './examples.component';
 import { TabsExampleComponent } from './tabs/tabs-example.component';
 import { TabExampleComponent } from './tabs/tab/tab-example.component';
+import { ListDefaultExampleComponent } from '~/app/examples/list/default/default';
```

Comment:
Consider using relative imports:
```suggestion
import { ListDefaultExampleComponent } from './list/default/default';
```